vulnfeeds/cmd/alpine/run_alpine_convert.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+OSV_PARTS_OUTPUT=parts/alpine_output
```

Comment:
Quote this string: https://google.github.io/styleguide/shellguide.html#quoting and also quote all of your variable usage to make it space-safe.